src/apps/backend/modules/account/account_service.py

```diff
@@ -1,13 +1,23 @@
+from datetime import datetime
+
+from pymongo import ReturnDocument
```

Comment:
It is not Ok for service layer to know the underlying storage. Only reader and write should know that, and they should return data objects in return.